hazelcast/src/main/java/com/hazelcast/jet/impl/client/protocol/task/JetGetJobSummaryListMessageTask.java

```diff
@@ -60,7 +60,9 @@ public Collection<Member> getTargets() {
     @SuppressWarnings("unchecked")
     @Override
     protected Object reduce(Map<Member, Object> map) throws Throwable {
+        rethrowInReduce(map, true);
         return map.values().stream()
+                .filter(item -> item instanceof List<?>)
```

Comment:
Again.

Reply:
This is needed, since you can have both MemberLeftException and real result in the map.